src/extensions/copypasta.py

```diff
@@ -0,0 +1,111 @@
+import os
+
+import aiofiles
+import arc
+import hikari
+
+from src.config import Colour
+from src.models import Blockbot, BlockbotContext, BlockbotPlugin
+
+plugin = BlockbotPlugin(name="Copypasta")
+
+MAX_EMBED = 4096
+MAX_MESSAGE = 6000
+
+
+async def load_text(file_name: str) -> list[list[str]]:
+    async with aiofiles.open(
+        f"./src/copypastas/{file_name}.txt", "r", encoding="utf-8"
+    ) as f:
+        text = await f.read()
+
+    messages: list[list[str]] = []
+    current_message: list[str] = []
+    message_len = 0
+
+    current_embed = ""
+    embed_len = 0
+
+    for line in text.splitlines(keepends=True):
+        line_len = len(line)
+
+        if line_len > MAX_EMBED:
+            # probably won't happen but you never know...
+            raise ValueError("line exceeds embed size limit")
+
+        # new embed
+        if embed_len + line_len > MAX_EMBED:
+            current_message.append(current_embed)
+            message_len += embed_len
+            current_embed = ""
+            embed_len = 0
+
+        # new message
+        if message_len + embed_len + line_len > MAX_MESSAGE:
+            if embed_len:
+                current_message.append(current_embed)
+            messages.append(current_message)
+
+            current_message = []
+            message_len = 0
+            current_embed = ""
+            embed_len = 0
+
+        current_embed += line
+        embed_len += line_len
+
+    # handle leftovers
+    if embed_len:
+        current_message.append(current_embed)
+    if current_message:
+        messages.append(current_message)
+
+    return messages
+
+
+@plugin.include
+@arc.slash_command("copypasta", "So tell me Frank!")
+async def gerry_command(
```

Comment:
Might want to rename the function to `copypasta_command` since we don't have /gerry, but instead use /copypasta.
```suggestion
async def copypasta_command(
```